src/V3Undriven.cpp

```diff
@@ -393,6 +446,7 @@ class UndrivenVisitor : public AstNVisitor {
     }
     virtual void visit(AstAssignW* nodep) override {
         VL_RESTORER(m_inContAssign);
+        VL_RESTORER(m_constHasZ);
```

Comment:
I think this is OK but would note it will also detect a Z on the left hand side.  But that's seems nonsensical, so this is ok, just a note for other similar cases.